changelog/changelog.ddoc

```diff
@@ -15,14 +15,15 @@ SUBNAV=$(SUBNAV_TEMPLATE
 SEARCH_OPTIONS_EXTRA=<option value="dlang.org/changelog" selected>Change
 Log</option>
 
-VERSION=
+STABLE_VERSION=
```

Comment:
Not sure I have the same understanding of STABLE_VERSION as you, so far we used that term to pin tools/compilers, but not for anything in our docs/targets.